autocertkit/ack_cli.py

```diff
@@ -440,6 +440,7 @@ def main(config, test_run_file):
     utils.log.info("Starting Test Runner from ACK CLI.")
     test_file, output = test_runner.run_tests_from_file(test_run_file)
 
+
```

Comment:
The travis-ci executes tools/checkpep8.sh to check the code, the log shows that these blank lines should be added. 
I will remove these blank lines.